code/datums/traits/good.dm

```diff
@@ -108,3 +108,19 @@
 	mob_trait = TRAIT_VORACIOUS
 	gain_text = "<span class='notice'>You feel HONGRY.</span>"
 	lose_text = "<span class='danger'>You no longer feel HONGRY.</span>"
+
+
+
+/datum/trait/hlumi_eyes
```

Comment:
/datum/trait/hlumi_eyes -->/datum/quirk/hlumi_eyes

Reply:
yeah that's probably what's fucking it